src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor

```diff
@@ -26,6 +26,7 @@
                       @oninput="HandleFilter"
                       @bind-Value:after="HandleClear"
                       Placeholder="Filter..."
+                      Title="Message filter"
```

Comment:
nit: Since this is the html global attribute `title` and not a razor component parameter `Title`, it is probably better to make it lower-case to make it clear(ish) which you're specifying (some components have a Title attribute, and in some of those cases its obsolete). To be clear, what you have here will work just fine since the html attributes are case-insensitive. Just a small thing for clarity :)


```suggestion
                      title="Message filter"
```